backend/src/openarchiefbeheer/destruction/constants.py

```diff
@@ -39,3 +39,6 @@ class InternalStatus(models.TextChoices):
     processing = "processing", _("processing")
     failed = "failed", _("failed")
     succeeded = "succeeded", _("succeeded")
+
+
+WAITING_PERIOD = 7  # Days
```

Comment:
Feels to me that this could be a configurable (DB) setting?

Reply:
Yes! The ticket didn't say about making it configurable, but i put it in this file so that it's easier to refactor if we want to configure it.